docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -71,49 +59,71 @@ The [`download-data.py` script](https://github.com/AlexsLemonade/OpenScPCA-analy
 
-We briefly cover some of the most common use cases below, but we encourage you to review all the options available to you.
+- You can list all possible arguments and their usage for the download data script with the following:
 
-You can list all the options for the download data script by running the following from the root directory of the repository:
+    ```sh
+    ./download-data.py --help
+    ```
 
-```sh
-./download-data.py --help
-```
+- You can run the download script with all default options to download all processed samples from the most recent release in `SingleCellExperiment` format:
 
-You can run the download script with all default options to download all processed samples from the most recent release in `SingleCellExperiment` format:
+    !!! warning
+        **The full data release download is quite large: over 35 GB for `SingleCellExperiment` format, and over 100 GB for `AnnData` format.
+        Please download the full data release with caution!**
 
-!!! warning
-    **The full data release download is quite large: over 35 GB for `SingleCellExperiment` format, and over 100 GB for `AnnData` format.
-    Please download the full data release with caution!**
+    ```sh
+    ./download-data.py
+    ```
 
-```sh
-./download-data.py
-```
+- If you prefer to work with `AnnData` files, use the `--format` option as follows:
 
-If you prefer to work with `AnnData` files, use the `--format` option as follows:
+    ```sh
+    ./download-data.py --format AnnData
+    ```
 
-```sh
-./download-data.py --format AnnData
-```
+- To download a specific release other than the most recent release, you can use the `--release` option as follows.
+Again, please download full data releases with caution, as they are quite large!
+    - First, you may wish to use the `--list-releases` flag to see all available releases, which are named in `YYYY-MM-DD` format based on their release date.
 
-To review what samples would be downloaded without performing the download yet, you can use the `--dryrun` option as follows:
+    ```sh
+    # First, see which releases are available for download
+    ./download-data.py --list-releases
 
-```sh
-./download-data.py --dryrun
-```
+    # Download SingleCellExperiment format files for a specific release
+    # For example, this downloads the 2024-05-01 release
+    ./download-data.py --release 2024-05-01
 
-If you're only working with a subset of the data, you can use the `--projects` or `--samples` to download select samples (note: these options are mutually exclusive):
+    # Download AnnData format files for a specific release, for example
+    ./download-data.py --release 2024-05-01 --format AnnData
+    ```
 
-```sh
-# use the --projects option
-./download-data.py --projects SCPCPXXXXXX
+- If you're only working with a subset of the data, you can use the `--projects` or `--samples` option to download select projects or samples, respectively (note that these options are mutually exclusive):
```

Comment:
Let's move the project/sample options before the release option? It will flow better into the symlink update option mentioned later as well.